src/master-playlist-controller.js

```diff
@@ -569,6 +569,23 @@ export class MasterPlaylistController extends videojs.EventTarget {
   }
 
   /**
+   * Convert the properties of an HLS track into an audioTrackKind.
+   *
+   * @private
+   */
+  audioTrackKind_(properties) {
+    let kind = properties.default ? 'main' : 'alternative';
+
+    if (properties.characteristics) {
+      if (properties.characteristics.indexOf('public.accessibility.describes-video') >= 0) {
+        kind = 'main-desc';
```

Comment:
In theory that's possible; I'd assume it would need to follow the same rules as, say, a CAPTIONS or SUBTITLES tracks:

https://tools.ietf.org/html/draft-pantos-http-live-streaming-20#section-4.3.4.1.1:
> All EXT-X-MEDIA tags in a Playlist MUST meet the following
>    constraints:
> 
>    o  All EXT-X-MEDIA tags in the same Group MUST have different NAME
>       attributes.
> 
>    o  A Group MUST NOT have more than one member with a DEFAULT
>       attribute of YES.
> 
>    o  All members of a Group whose AUTOSELECT attribute has a value of
>       YES MUST have LANGUAGE [RFC5646] attributes with unique values.
> 

So you could tell them apart by `name` (`label`) and `language`. But it wouldn't change the audio track's `kind`. Is that what you were wondering about?